examples/example_utils.cpp

```diff
@@ -1,6 +1,12 @@
 #include <vector>
 #include <stdexcept>
 
+#include <stdarg.h>
+
+#ifdef MIP_ENABLE_LOGGING
+#include <mip/mip_logger.h>
+#endif
```

Comment:
I think files should just include the logging file regardless of the flag, and have mip_logger.h (mip_logging.h?) file do the check. This would cut down on verbosity a bit if lots of files end up using logging.

Reply:
So I think this is something specific to the examples. The reason for this include guard is to prevent linker errors if we accidentally tried to initialize logging without enabling it in the examples.

Most users should have the files installed, so it would be obvious that they could not include `mip/mip_logger.h` if `MIP_ENABLE_LOGGING` was not set.

However, if someone was building out of the source tree, this might be confusing and annoying.

I could add a macro that we define as nothing if `MIP_ENABLE_LOGGING` is off for `mip_logging_init`, something like `MIP_LOGGING_INIT`